pkg/apis/configuration/validation/validation.go

```diff
@@ -156,6 +159,41 @@ func validateUpstreams(upstreams []v1alpha1.Upstream, fieldPath *field.Path, isP
 	return allErrs, upstreamNames
 }
 
+var validVariableParams = map[string]bool{
```

Comment:
it is a global variable, so the name must be more specific. Consider changing to `validNextUpstreamParams`